### PR TITLE
Feature/manual prettify

### PR DESCRIPTION
**Pull request by honestbonsai**
Continuation of #1087 for resolution of #1085 

This PR is for all of my manual edits. Trimmed down the remaining ~300 lint errors left over from the auto prettify to about 40.

Pretty much only did `no-unused-vars` and a couple of other minor ones that don't involve any logical  or graphical refactors to lean on the safe side.

The remaining lint errors are mostly `react/no-direct-mutation-state`, `react/no-string-refs`, `react/no-unescaped-entities`, etc.

EDIT: You can refer to this [commit range](https://github.com/trufflesuite/ganache/pull/1088/files/8b9475018e3f731d65a26a3c1f468ac15ade6651..541934518d7259ad30b06a45d8a6d9deca8ed5de) for ease of reading. I omitted 8b94750 (the auto prettify)

### CODE REVIEW

**Comment by adrianmcli** [comment]
Can you rebase this off of `develop`?

**Comment by adrianmcli** [comment]
It should be much fewer files if it was rebased off develop. At least that's what I believe.

**Comment by honestbonsai** [comment]
The rebase is actually quite odd, you can see a test I did here:
https://github.com/trufflesuite/ganache/compare/feature/manual-prettify-rebase-test

**Comment by adrianmcli** [comment]
@honestbonsai I think you should do a squash after. But either way, I think this is good. Execute the rebase please.

**Comment by hyperh** [comment]
@seesemichaelj It's been rebased (thx for the help @adrianmcli) if you want to take a quick look at the changes.

Just waiting for the build to complete.

**Comment by honestbonsai** [comment]
@seesemichaelj It's been rebased and squashed (thx for the help @adrianmcli). You can take a look at the changes.

Just waiting for the build to finish.

**Comment by adrianmcli** [comment]
Can't wait for this to be merged.

File: src/common/redux/middleware/analytics/engines/GoogleAnalytics.js [hunk 1]
```diff
@@ -66,35 +66,35 @@ function SystemErrorEvent(category, detail) {
...
 
-function RPCRequestStartedEvent(payload) {
-  return {
-    category: "rpc",
-    action: "started",
-    label: payload.method || "(unknown method)",
-    value: payload.params ? payload.params.length : 0,
-  };
-}
+// function RPCRequestStartedEvent(payload) {
```
Comment (seesemichaelj) [inline src/common/redux/middleware/analytics/engines/GoogleAnalytics.js]: thanks for reading between the lines on this one :)

**Pull request merged**